### PR TITLE
DO NOT MERGE - TEST PR

### PR DESCRIPTION
**Pull request by dospromptman**
See bug #1812

### CODE REVIEW

**Comment by dospromptman** [comment]
@jvarese what up

**Comment by jvarese** [comment]
@dospromptman - did you get email notification for this?

**Comment by jvarese** [comment]
@davidhabib @kbromer @jlantz - I am not getting email notifications from github when I am @mentioned. Wondering if you have any ideas as to why. All my settings are set correctly under Emails and Notifications, to receive everything. Any weird glitches you've ever seen or remedied?

**Comment by kbromer** [comment]
Spam folder? Dunno otherwise.

File: src/classes/REL_Relationships_TEST.cls [hunk 1]
```diff
@@ -859,5 +859,85 @@ private class REL_Relationships_TEST {
             REL_RelationshipsViewer_CTRL.RelationshipInfo ri = REL_RelationshipsViewer_CTRL.getRelationshipInfo();
             system.assertNotEquals(null, ri);                       
         }
-    }   
+    }
+
+    public static testMethod void testDeletingMutuallyReferentialRelationshipsDoesNotPreventDeleteOfReciprocals() {
```
Comment (LurchTheButler) [inline src/classes/REL_Relationships_TEST.cls]: I suggest to annotate test methods with `@IsTest` for better readability.

Comment (dospromptman) [inline src/classes/REL_Relationships_TEST.cls]: Thanks, Lurch.  I agree.  I'm rejecting this pull request until this issue has been remedied.

**Pull request closed without merging**